setup.py

```diff
@@ -165,6 +164,8 @@ def build_extensions(self):
     ),
 ]
 
+if os.environ.get("USING_RTD", False):
+    ext_modules = []
```

Comment:
Am thinking to use this: https://docs.readthedocs.io/en/stable/guides/environment-variables.html

Reply:
@trbromley how come? Could it be useful to be able to compile the documentation locally even without the C++ requirements?

Reply:
Oh I suppose you could always set this locally! Perhaps you could make the env variable more general? `LQ_MOCK_DOCS`?

Reply:
Nice idea!